libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Schemas/Actions/Teams.SendMessagingExtensionConfigQuerySettingUrlResponse.schema

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
     "$role": "implements(Microsoft.IDialog)",
     "title": "Send query link response",
-    "description": "Query link requests have activity.name='composeExtension/queryLink'.",
+    "description": "Query link requests have activity.name='composeExtension/setting'.",
     "type": "object",
```

Comment:
This file has an amazingly long $kind--is that because it corresponds to a Teams class?  Also, this doesn't feel like a description.  I don't think this is what it does is it?